charts/nexus3/files/configure.sh

```diff
@@ -1,6 +1,14 @@
 #!/usr/bin/env bash
 set -eu
 
+function error() {
+  msg="ERROR $*"
+  >&2 echo "$msg"
+  echo "$msg" > "$TERMINATION_LOG"
+  exit 1
+}
+
+TERMINATION_LOG=${TERMINATION_LOG:-/dev/termination-log}
```

Comment:
```suggestion
function error() {
  msg="ERROR: $*"
  >&2 echo "${msg}"
  echo "${msg}" > "${TERMINATION_LOG}"
  exit 1
}

TERMINATION_LOG="${TERMINATION_LOG:-/dev/termination-log}"
```

Reply:
new commit pushed.
thanks for the fantastically fast review!